lizard_auth_client/tests.py

```diff
@@ -1071,6 +1089,107 @@ def test_organisation_index_view_superuser(self):
         response = views.ManageOrganisationIndex.as_view()(request)
         self.assertEqual(response.status_code, 200)
 
+    def get_new_user_sso_mock(self, override_user_data=None, status_code=201):
```

Comment:
Als je de methode prefixt met een underscore is het *nog* wat duidelijker dat het geen testmethode is. Zelfde met de methode verder naar beneden.